document/FileFormat.mdk

```diff
@@ -2696,31 +2708,385 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
+"recommended" means that the property should be present, and "absent" means that the property shall be absent.
+
+In this table, a "variable name" may be any of the following, unless otherwise specified:
+
+* A simple variable name.
+* An object property reference.
+* An array element reference.
+* A reference to the current object.
+* Any combination of these.
+
+~ IsoNumberedExample { @IsoNumberedExample=1 }
+Examples of valid "variable names" in C++:
+
+* `count`
+* `str->length`
+* `values[0]`
+* `this`
+* `this->size`
+* `this->car->wheels[0]`
+
+~
+
+In the following table, whenever a "value" is mentioned, it means a string representation of the
+value.
+
+~ IsoNumberedExample
+Examples of valid "values":
+
+* A integer value of `2` would be represented as `"2"`.
+* A string value of `"abc"` would be represented as `"\"abc\""`.
+* A Boolean value of `true` would be represented as `"true"`.
+
+~
+
+~ Center
+Table [#annotatedCodeLocation-target].1: Interpretation of `target`, `variables`, and `parameters`
+
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| **`kind`**       | **`target`**                | **`parameters`**            | **`variables`**             |
++:-----------------+:----------------------------+:----------------------------+:----------------------------+
+| `alias`          | Recommended: The            | Recommended: The            | Optional: The name and      |
+|                  | name of the alias being     | name of the variable        | current value of the        |
+|                  | created. If multiple        | being aliased. Only a       | variable being aliased.     |
+|                  | aliases are created in      | simple variable name        |                             |
+|                  | the same statement, a       | shall be specified here.    |                             |
+|                  | separate                    |                             |                             |
+|                  | `annotatedCodeLocation`     |                             |                             |
+|                  | object shall be created     |                             |                             |
+|                  | for each alias that the     |                             |                             |
+|                  | tool creating the log       |                             |                             |
+|                  | wishes to represent.        |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `assignment`     | Recommended: The            | Recommended: The            | Optional: the names and     |
+|                  | name of the variable        | value assigned to           | values of the variables     |
```

Comment:
Fixed.

---
In reply to: [77678873](https://github.com/sarif-standard/sarif-spec/pull/264#discussion_r77678873) [](ancestors = 77678873)